src/patternfly/layouts/Grid/examples/grid-responsive-gutters-example.hbs

```diff
@@ -0,0 +1,62 @@
+{{#> grid grid--modifier="pf-m-gutter pf-m-gutter-3xl-on-lg"}}
+  {{#> grid-item grid-item--modifier="pf-m-6-col"}}
+      <strong>.pf-m-gutter.pf-m-gutter-3xl-on-lg</strong> - Item 1
+  {{/grid-item}}
+  {{#> grid-item grid-item--modifier="pf-m-6-col"}}
+      Item 2
+  {{/grid-item}}
+  {{#> grid-item grid-item--modifier="pf-m-3-col"}}
+      Item 3
+  {{/grid-item}}
+  {{#> grid-item grid-item--modifier="pf-m-3-col"}}
+      Item 4
+  {{/grid-item}}
+  {{#> grid-item grid-item--modifier="pf-m-3-col"}}
+      Item 5
+  {{/grid-item}}
+  {{#> grid-item grid-item--modifier="pf-m-3-col"}}
+      Item 6
+  {{/grid-item}}
+{{/grid}}
+<br><br>
```

Comment:
In between these examples could you add a description about each variation, looking at this seems like they're all one example and its a little confusing

<img width="886" alt="Screen Shot 2019-05-13 at 10 32 16 AM" src="https://user-images.githubusercontent.com/20118816/57629801-68fe7300-756a-11e9-9096-b899ce0b9a06.png">